Duanaire_Finn/PoemXI.trig

```diff
@@ -68,7 +68,8 @@ _:missing-8d30e759
 <#Baoiscne-5b44daaa>
     a foaf:Person ;
     irishRel:accName "Baoiscne" ;
-    irishRel:nomName "Baoiscne" .
+    irishRel:nomName "Baoiscne" ;
+    owl:sameAs <http://example.com/Duanaire_Finn/PoemXXXVII.trig#Baoisgne>.
```

Comment:
I agree with the identification but, as noted previously, I think this can be merged with `<#Baoiscne>`. Apologies - I've accidentally reviewed each commit separately!

Reply:
The `owl:sameAs` link in this change is still needed, I think?